modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -29,7 +29,14 @@ For more information about how to install Helm, see the Helm installation docume
 ** Google Kubernetes Engine (GKE)
 ** Amazon EKS
 ** Azure Kubernetes Service (AKS)
-* Not installed Istio in the cluster
+* https://istio.io/docs/setup[Install Istio] 1.4.x in your cluster with the following configurations: 
+. Enable policy control by configuring the flags:
+** `--set values.global.disablePolicyChecks=false`
+** `--set pilot.policy.enabled=true`
++
+For more information, see https://istio.io/docs/tasks/policy-enforcement/enabling-policy/[Policy Enforcement]. 
+. Enable API Analytics by setting the `disableMixerHttpReports` flag to *false*: 
```

Comment:
I'm wondering if this should be a regular bullet instead of a step.

Reply:
This applies to the Istio installation. So this is a sub-step.